protocol/messenger_communities.go

```diff
@@ -1205,6 +1205,9 @@ func (m *Messenger) SpectateCommunity(communityID types.HexBytes) (*MessengerRes
 		return nil, err
 	}
 
+	// sync community
+	m.asyncRequestAllHistoricMessages()
```

Comment:
This looks a bit too much for just spectating a community 🤔 
We should be able to just request history for this specific community, right? 

Though it's not a big deal.